instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_bedrock.py

```diff
@@ -1980,12 +2112,16 @@ def invoke_model_with_response_stream_tool_call(
 
     content = []
     content_block = {}
+    # used only by anthropic claude
     input_json_buf = ""
+    # used only by amazon nova
+    tool_use = {}
     for event in response_0["body"]:
         json_bytes = event["chunk"].get("bytes", b"")
         decoded = json_bytes.decode("utf-8")
         chunk = json.loads(decoded)
 
+        # anthropic claude
```

Comment:
Is it possible to push these branches similarly `llm_model_config`?

Reply:
Done